python/tls/README.md

```diff
@@ -0,0 +1,3 @@
+## Python TLS Package for Fybrik Components
+The versions of this library do not correspond to the
+versions of the Fybrik repository.
```

Comment:
why do we need a separate version here ? 
If we use a separate versioning here, please add it into the backward compatibility doc.

Reply:
I think we do. Fybrik has its own considerations for bumping versions that are unrelated to the python library (it relates to the module code if any). On the other hand, the go library  depends on fybrik. The logging mentioned in https://github.com/fybrik/fybrik/blob/master/docs/BackwardCompatibility.md#format-of-fybrik-log-files seems to refer to the go logging library as for python we have this [statement](https://github.com/fybrik/fybrik/tree/master/python/logging) in the README.d file. Let talk to clarify this.